cmd/kubectl-testkube/commands/tests/create.go

```diff
@@ -61,7 +63,8 @@ func NewCreateTestsCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&file, "file", "f", "", "JSON test file - will be read from stdin if not specified, look at testkube.TestUpsertRequest")
-	cmd.Flags().StringSliceVar(&tags, "tags", nil, "--tags 1,2,3")
+	cmd.Flags().StringVar(&name, "name", "", "Set/Override test name")
+	cmd.Flags().StringSliceVar(&tags, "tags", nil, "--tags tag1,tag2,tag3")
```

Comment:
Shouldn't it be like in other places `comma separated list of tags: --tags tag1,tag2,tag3"`?

Reply:
yes - good point - was doing find/replace and missed this one - done @vsukhin